packages/web-react/DEPRECATIONS-v2.md

```diff
@@ -0,0 +1,10 @@
+# Deprecations List
+
+Following deprecations will be removed in version 2 of the _spirit-web-react_ package
+
+> Please follow these steps to safely upgrade your Design System to Spirit Design System v2 components.
+
+## Add the `danger` icon between your project's assets
```

Comment:
```suggestion
Following deprecations will be removed in version 2 of the _spirit-web-react_ package.

> Please follow these steps to safely upgrade your design system to Spirit Design System v2 components.

## Add the `danger` icon to your project's assets
```

Reply:
Heh, line 5 is just a copy&paste from the `MIGRATIONS-v1.md` 🤷

Reply:
Fixed.